.github/workflows/ci.yml

```diff
@@ -438,6 +438,7 @@ jobs:
       - uses: ./.github/actions/setup
         with:
           cache: restore
+          cleanup-node: true
```

Comment:
is this the equivalent of `docker system prune`?

Reply:
it does more things, see `.github/bin/free-disk-space.sh` for the impl.